src/services/StatisticsService.js

```diff
@@ -0,0 +1,22 @@
+import instance from '../utilities/axios/axios';
+
+export async function getStatistics() {
```

Comment:
why did you name it starting with `get`? all other functions are named with `fetch`